package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.0.1",
   "private": true,
   "devDependencies": {
+    "bourbon": "^4.2.7",
```

Comment:
I think dev dependencies is fine since we have to commit built code to the repo for github pages. Can we move these out of "dependencies"?